lib/less/rails/railtie.rb

```diff
@@ -16,7 +16,7 @@ class Railtie < ::Rails::Railtie
       end
 
       initializer 'less-rails.before.load_config_initializers', :before => :load_config_initializers, :group => :all do |app|
-        app.assets.register_preprocessor 'text/css', ImportProcessor
+        app.assets.register_preprocessor 'text/css', ImportProcessor if app.assets
```

Comment:
@simi 
I fix the code but some tests didn't pass  at Rails 4.2 .

```
1) Failure:
initialization#test_0003_must register our import pre processor [/Users/suzuken/Documents/suzan2go/less-rails/test/cases/railtie_spec.rb:43]:
```
I think `app.config.assets` ( class `Sprockets::Railtie::OrderedOptions`) does not have  method `register_preprocessor`.

So I think we shoud do it like:
```
sprockets_env = app.assets || Sprockets 
sprockets_env.register_preprocessor 'text/css', ImportProcessor
```

https://github.com/reactjs/react-rails/blob/c463b8eff14806d5d4249520a7588ca1762e0f80/lib/react/rails/engine.rb#L5